run.sh

```diff
@@ -77,8 +77,6 @@ if [ -f "/etc/ipsec.d/xl2tpd.conf" ]; then
 	cp -f /etc/ipsec.d/xl2tpd.conf /etc/xl2tpd/xl2tpd.conf
 fi
 
-echo "Starting XL2TPD process..."
 mkdir -p /var/run/xl2tpd
-/usr/sbin/xl2tpd -c /etc/xl2tpd/xl2tpd.conf
 
-ipsec start --nofork\
+exec /usr/bin/supervisord -c /supervisord.conf
```

Comment:
The config file can maybe be kept in the default location `/etc/supervisor/conf.d/`?